hr_employee_ciam_client/models/client_config.py

```diff
@@ -87,10 +86,10 @@ def create_request_history(self, method, url, response, payload=False,
     def get_headers(self):
         tracking_id = pycompat.text_type(uuid.uuid1())
         headers = {
-            'x-amf-mediaType': "application/json",
+            'Content-Type': "application/json",
             'AF-TrackingId': tracking_id,
-            'AF-SystemId': "AF-SystemId",
-            'AF-EndUserId': "AF-EndUserId",
+            'AF-SystemId': "CRM",
+            'AF-EndUserId': "*sys*",
```

Comment:
Check Fredriks comment on this
https://github.com/vertelab/odoo-hr/pull/34#issuecomment-744648576